packages/@coorpacademy-translate/src/translate.js

```diff
@@ -16,21 +16,25 @@ function getTemplate(locales, key, count) {
   return getOr(regularTemplate, `${key}_plural`, locales);
 }
 
-const createTranslate = locales => (key, data) => {
-  const template = getTemplate(locales, key, get('count', data));
-  if (!isString(template)) {
-    throw new Error(`Key ${key} not found!`);
-  }
+const createTranslate =
+  (locales, isAllowFalsy = false) =>
+  (key, data) => {
+    const template = getTemplate(locales, key, get('count', data));
+    if (!isString(template) && !isAllowFalsy) {
+      throw new Error(`Key ${key} not found!`);
+    }
 
-  return replace(
-    interpolation,
-    (token, value) => {
-      const _value = trim(value);
-      return has(_value, data) ? get(_value, data) : token;
-    },
-    template
-  );
-};
+    return !isString(template)
```

Comment:
tip: if you want to minimize change you can just `if (!isString(template) return key`

Reply:
if fact, you can simplify the implementation. 

Pseudocode:
```
 if !isString(template):
   if throwIfMissing: throw
   return key

 return replace 
```